### PR TITLE
Clean up of network/telemetry guides in prep for translation

### PR DESCRIPTION
**Pull request by krisdale**

### CODE REVIEW

File: source/clear-linux/guides/network/vnc.rst [hunk 1]
```diff
@@ -362,11 +363,11 @@ On Fedora:
 
    # dnf install tigervnc
 
-On Windows:
+On Windows\*:
```
Comment (MCamp859) [inline source/clear-linux/guides/network/vnc.rst]: Branding changes needed above:   
Ubuntu\*, Linux Mint\*
Fedora\*

File: source/clear-linux/guides/network/vnc.rst [hunk 2]
```diff
@@ -408,7 +409,7 @@ SSH into your host and launch VNC
 
       #. Launch Putty.
       #. Under the :guilabel:`Category` section, select :guilabel:`Session`.
-         See Figure 1.
+         See figure 1.
```
Comment (MCamp859) [inline source/clear-linux/guides/network/vnc.rst]: Is it our convention to refer to figures like this?

File: source/clear-linux/guides/telemetrics/telem-guide.rst [hunk 3]
```diff
@@ -358,12 +360,12 @@ using the :ref:`bare-metal-install-server` getting started guide and:
...
-   :guilabel:`PostgreSQL` database password will be updated.
+   password, retype it to verify the new password and the :guilabel:`PostgreSQL`
+   database password will be updated.
 
 #. After the installation is complete, you can use your web browser to view the
-   new server by opening the browser on the system and typing in ``localhost``
-   in the address bar.  You should see a web page similar to the one shown in
+   new server by opening the browser on the system and typing in localhost
+   in the address bar. You should see a web page similar to the one shown in
    figure 1:
```
Comment (MCamp859) [inline source/clear-linux/guides/telemetrics/telem-guide.rst]: "Figure 2 below."   (cap F, 2, no colon)  
Also change caption to "Figure 2" in line 374.

File: source/clear-linux/guides/network/vnc.rst [hunk 4]
```diff
@@ -1,28 +1,26 @@
...
 
 To configure VNC to work on your |CL| host, install these bundles:
 
-* `desktop-autostart`: Installs :abbr:`GDM (Gnome Desktop Manager)`, sets
+* :command:`desktop-autostart`: Installs :abbr:`GDM (Gnome Desktop Manager)`, sets
   it to start automatically on boot, and installs TigerVNC Viewer.
-* `vnc-server`: Installs the TigerVNC server.
+* :command:`vnc-server`: Installs the TigerVNC server.
 
 Follow these steps:
```
[the remark this answers is not in the document]
Comment (krisdale) [reply source/clear-linux/guides/network/vnc.rst]: Good catch!

File: source/clear-linux/guides/network/vnc.rst (hunk 2 above)
Comment (krisdale) [reply source/clear-linux/guides/network/vnc.rst]: Meaning the capitalization of "figure"? Yes, standardizing after what @mvincerx  does.

**Comment by krisdale** [comment]
@mvincerx  - I've made the changes requested by @MCamp859. I believe this is good to go.

**Comment by mvincerx** [comment]
Hold please. I'm rebasing and pushing to this branch.

File: source/clear-linux/guides/network/vnc.rst (hunk 2 above)
Comment (mvincerx) [reply source/clear-linux/guides/network/vnc.rst]: Um, well, I capitalize references to figures. Example: See Figure 1. I followed those who preceded me, so I'm can't explain why!

**Comment by mvincerx** [comment]
Rebased. LGTM.

File: source/clear-linux/guides/network/vnc.rst (hunk 2 above)
Comment (MCamp859) [reply source/clear-linux/guides/network/vnc.rst]: That's why I asked... Uppercase refs to figs or tables is the BKM going back thru time, lol. Also, lowercase refs to figs or tables is not the convention described in our [guide](https://clearlinux.org/documentation/clear-linux/reference/collaboration/structure-formatting#images). Just want to be consistent...

**Pull request merged**